yarn.lock

```diff
@@ -2810,6 +2810,11 @@
     tsoa "^3.4.0"
     ws "^8.3.0"
 
+"@shapeshiftoss/caip@^4.0.1":
```

Comment:
This should go away after caip package v5.x.x published correctly

Reply:
Seems like it won't until all of these dependent PRs are in.